### PR TITLE
Restructure the files and add first required repo texts

### PR DESCRIPTION
**Pull request by kideh88**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

- Add basic MD files
- Fix contributors
- Add simple  start sh and streamlit app

### CODE REVIEW

File: DRCODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# This file defines which domain owns what parts of this repository.
+# This repository is a central place for defining Jenkins Jobs and Jarvis Suites to orchestrate
+# build and test workflows for pull requests and pipelines.  As such, it is expected
+# that this repository will have many owners and some shared areas (reusable macros, etc.)
+#
+# Please see DataRobot repo ./tools/code_owners.py tool to find which particular file
```
Comment (raulpineda) [inline DRCODEOWNERS]: Is this supposed to be present in public repos?

File: src/constants.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+API_URL = '{base_url}/predApi/v1.0/deployments/{deployment_id}/predictions'  # noqa
```
Comment (raulpineda) [inline src/constants.py]: This constant doesn't seem to be used in the application. Is there an intention to do so at some point or can we remove it?

**Review by raulpineda — APPROVED** [review]

File: src/constants.py (hunk 2 above)
Comment (kideh88) [reply src/constants.py]: I was wondering if we should provide a basic predict function, but on second thought we don't really know what users would try to include in their request :thinking:

File: DRCODEOWNERS (hunk 1 above)
Comment (kideh88) [reply DRCODEOWNERS]: Good point, removing it again

**Pull request merged**